samples: bluetooth: fast_pair: Change advertising mode when paired (#8330)
nrfconnect/sdk-nrf: samples/bluetooth/peripheral_fast_pair/README.rst

   .. note::
       The advertising is disabled while the Fast Pair Provider is connected to a Bluetooth Central.

>        The discoverable advertising is automatically switched to the not discoverable advertising with the show UI indication mode after a 10-minute timeout.

Reviewer: Maybe:
```
The discoverable advertising is automatically switched to the not discoverable advertising with the show UI indication mode in the following cases:
- after 10 minutes of continuous discoverable advertising
- after a Central connects and successfully bonds
```

>        Advertising mode is automatically switched to the not discoverable advertising with the show UI indication mode after a Bluetooth Central successfully pairs.

Reviewer 2: ```suggestion
       The advertising mode is automatically switched to the not discoverable advertising with the show UI indication mode after a Bluetooth Central successfully pairs.
```

>        The advertising is disabled while the Fast Pair Provider is connected to a Bluetooth Central.
>        The discoverable advertising is automatically switched to the not discoverable advertising with the show UI indication mode after a 10-minute timeout.
>        The advertising mode is automatically switched to the not discoverable advertising with the show UI indication mode after a Bluetooth Central successfully pairs.

Reviewer 3: This new line reads like contradicting line 96, because "while" also includes case now added in line 98. What about:
```suggestion
       The advertising mode automatically switches to the not discoverable advertising with the show UI indication mode after a Bluetooth Central successfully pairs.
       If the discoverable advertising is toggled, it automatically switches to the not discoverable advertising with the show UI indication mode after a 10-minute timeout.
       
```

Author: The 96 line is still true. Advertising is disabled and the advertising mode is switched. Of course, the only consequence of switching advertising mode while advertising is disabled (device is connected) is changing the led effects. Actual advertising in new mode starts after the device disconnects.

>    .. note::
>        The advertising is disabled while the Fast Pair Provider is connected to a Bluetooth Central.
>        The discoverable advertising is automatically switched to the not discoverable advertising with the show UI indication mode after a 10-minute timeout.
>        The advertising mode is automatically switched to the not discoverable advertising with the show UI indication mode after a Bluetooth Central successfully pairs.

Reviewer 3: New proposal:
```suggestion
   .. note::
       The Bluetooth advertising is active only until the Fast Pair Provider connects to a Bluetooth Central.
       After the connection, you can still switch the advertising modes, but the switch will come into effect only after disconnection.
       The discoverable advertising is automatically switched to the not discoverable advertising with the show UI indication mode in the following cases:
       
       * After a 10-minute timeout.
       * After a Bluetooth Central successfully pairs.
```
@alstrzebonski , @MarekPieta , would this work?

Author: For me it's ok.

Author: Or maybe `* After 10 minutes of active advertising`? That would be more accurate.

      .. note::
         Do not remove Bluetooth bond information related to the Fast Pair Provider.

      After disconnection, the provider automatically switches from the discoverable advertising to the not discoverable advertising with the show UI indication mode.

>       **LED 3** is blinking rapidly.

Author: Yes they are but it happens earlier - after successful pairing. Not after disconnection.

   a. Go to :guilabel:`Settings` > :guilabel:`Google` > :guilabel:`Devices & sharing` (or :guilabel:`Device connections`) > :guilabel:`Devices` > :guilabel:`Saved devices`.
   #. Verify that the paired device is appearing on the list.

#. Make sure that the Fast Pair Provider is in the Fast Pair not discoverable advertising show UI indication mode.

>    **LED 3** must blink rapidly.

Reviewer: I would move this point right after the one about disconnecting Android and simplyfiy it, e.g.:
```
After disconnection, the provider will automatically switch from discoverable advertising to non-discoverable advertising show UI indication.
The LED 3 starts to blink rapidly.
```

Reviewer 3: Maybe in line 189/190:
```
After disconnection, the provider automatically switches from the discoverable advertising to the non discoverable advertising with the show UI indication mode.
The LED 3 starts to blink rapidly.
```
and then here in line 198:
```
#. If you want to test the Fast Pair not discoverable advertising hide UI indication mode, press **Button 1**.
```

Author: I would do `**LED 3** is blinking rapidly.`. It starts blinking rapidly after successful pairing.

>    **LED 3** should be blinking rapidly.

Reviewer 2: ```suggestion
   **LED 3** must blink rapidly.
```

>    If you want to test the Fast Pair not discoverable advertising hide UI indication mode, press **Button 1**.

Reviewer 3: This is not clear. How one can make sure that the FPP is in the right mode? What if LED 3 is not blinking rapidly? Is this the result of the previous step, that is adding the FPP to the Android device list?

Reviewer 2: This is a good point @greg-fer .

Also, My initial thought was that after Steps 1 and 2, the FPP is in the not discoverable show UI indication advertising mode. But I think that is not the case. 
Isn't it true that The FFP is in this mode only after FPP is added on the second device (after step 4) because we have requirement that - _Testing not discoverable advertising requires using at least two Android devices registered to the same Google account. The first one needs to be bonded with Fast Pair Provider before the not discoverable advertising can be detected by the second one._ : http://developer.nordicsemi.com/nRF_Connect_SDK_dev/doc/PR-8330/nrf/samples/bluetooth/peripheral_fast_pair/README.html#not-discoverable-advertising-requirements .

Author: Yes, as a consequence of previous step, advertising mode should switch to `not discoverable advertising show UI indication mode` which is denoted by LED 3 blinking rapidly. User should make sure that it really happened (it will happen only if Provider successfully pairs in previous step).

Author: If LED 3 didn't blink rapidly, advertising mode can be toggled by Button 1, but that won't be the case if previous step is successful.
